### ISSUE TITLE
Horizontal day view

### ISSUE THREAD

**Issue by sagar-ganatra** [angular-ui/ui-calendar#278]
Is it possible to display the day view horizontally instead of displaying it vertically?

**Comment by Sefriol** [comment]
I doubt it's possible without altering the source code.

**Comment by sagar-ganatra** [comment]
Do you know any other library that supports calendar day view in the horizontal format? I have used UI calendar in the past and this has come up as a requirement in a new project.

**Comment by Sefriol** [comment]
Not really. http://fullcalendar.io/timeline/ might be something to look for, but it isn't available yet.

Best bet would be to edit https://github.com/arshaw/fullcalendar/blob/master/src/basic/BasicView.js
yourself. It might take some time to achieve what you want, but fastly looking into it, it doesn't seem so hard.
